assets/js/metrics_live/index.js

```diff
@@ -220,7 +220,7 @@ class Summary {
       title: options.title,
       width: options.width,
       height: options.height,
-      tzDate: ts => uPlot.tzDate(new Date(ts * 1e3)),
+      tzDate: ts => uPlot.tzDate(new Date(ts)),
```

Comment:
if you're not using the timezone parameter of `uPlot.tzDate`, it would be cheaper to just do `tzDate: ts => new Date(ts)`.

alternatively, you can provide the ms timestamps in decimal format and avoid defining `tzDate` at all.

Reply:
Thanks @leeoniya! I'll tweak this a bit further.